ci/nightly/pipeline.template.yml

```diff
@@ -10,28 +10,39 @@
 # Default priority, just set to be explicit
 priority: 0
 
+env:
+  CI_BUILD_WITH_BAZEL: 1
+  CI_BAZEL_REMOTE_CACHE: "https://bazel-remote.dev.materialize.com"
+
 steps:
   - group: Builds
     key: builds
     steps:
       - id: build-x86_64
-        label: Build x86_64
+        label: ":bazel: Build x86_64"
         command: bin/ci-builder run stable bin/pyactivate -m ci.test.build
         inputs:
           - "*"
+          - "WORKSPACE"
+          - ".bazelrc"
+          - "**/*.bzl"
```

Comment:
Doesn't `inputs: *` already mean everything? If so, then I don't think you need to explicitly list out the Bazel files. Here and the several jobs below.